docs/operator-manual/declarative-setup.md

```diff
@@ -653,18 +653,12 @@ stringData:
       "execProviderConfig": {
         "command": "argocd-k8s-auth",
         "env":
```

Comment:
Think this is invalid object notation in JSON. Missing `{}` embracing the fields, and commas delimiting each field of the object.

Reply:
long week @Ziphone, I had YAML on the mind.  Updated PR